### PR TITLE
[PAN-3111] Moves getTransactionReceipt from eea to priv

### PR DESCRIPTION
**Pull request by josh-richardson**
Renames EeaGetTransactionReceipt to PrivGetTransactionReceipt

The relevant PR in Pantheon and the issue:
PegaSysEng/pantheon#1927
https://pegasys1.atlassian.net/secure/RapidBoard.jspa?rapidView=57&projectKey=PAN&modal=detail&selectedIssue=PAN-3111

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,10 +40,10 @@
     "ethereumjs-tx": "^1.3.7",
     "lodash": "^4.17.15",
     "rlp": "^2.2.3",
-    "web3": "^1.2.0"
+    "web3": "^1.2.0",
+    "axios": "^0.18.1"
```
Comment (iikirilov) [inline package.json]: How come this has moved?

Comment (iikirilov) [reply package.json]: nevermind - es-lint complains because this is not a dev-dependency

**Comment by josh-richardson** [comment]
@MadelineMurray good call. I've updated, but some of the integration tests still fail here as pantheon-quickstart is still outdated.

**Comment by josh-richardson** [comment]
Quickstart has now been updated and all of the integration tests now pass. This can be merged.

**Pull request merged**